crates/loans/src/benchmarking.rs

```diff
@@ -150,6 +150,34 @@ pub mod benchmarks {
 
     use super::*;
 
+    #[benchmark]
+    pub fn on_initialize(u: Linear<1, 2>) {
```

Comment:
I wanted to parameterize `on_initialize` on the number of markets, but since you pointed out that several extrinsics iterate over markets, it'd be cleaner to have a separate PR for this parameterization.

Reply:
This one is easier to address though since on_initialize uses a dynamic weight. Even when we eventually set an upperbound it's still better to have on_initialize return the actual weight rather than the upper bound (especially since this is called every block)